threddsTest/build.info

```diff
@@ -0,0 +1,2 @@
+../wmsConfig.xml
+../idd
```

Comment:
This should pull in all of what's used in the idd and thredds directory (in order), then just need to override the top-level catalog so the pre-processed catalog/notebook can be included:
```suggestion
../idd
../thredds
../wmsConfig.xml
```

Reply:
thanks, I see how that works now-- that is nicer. Fixed in 2edb0e56f7214270f90be90b5eaa79ad26b4230c